doc/overview.rst

```diff
@@ -5,17 +5,37 @@ SmartDashboard
 
 ``SmartDashboard`` is an add-on to SmartSim that provides a dashboard to help users understand
 and monitor their SmartSim experiments in a visual way. Configuration, status, and logs
-are available for all launched entities within an experiment for easy inspection.
+are available for all launched entities within an experiment for easy inspection,
+along with memory and client data per shard for launched orchestrators.
 
 A ``Telemetry Monitor`` is a background process that is launched along with the experiment
```

Comment:
what about:

A Telemetry Monitor is a background process launched alongside the experiment. It is responsible for generating the data displayed by SmartDashboard. The Telemetry Monitor can be globally disabled by setting the environment variable SMARTSIM_FLAG_TELEMETRY to 0 using the command export SMARTSIM_FLAG_TELEMETRY=0. When disabled, SmartDashboard will not display entity status data in the dashboard or to stdout. To re-enable it, set the SMARTSIM_FLAG_TELEMETRY environment variable to 1 with export SMARTSIM_FLAG_TELEMETRY=1. ~ For more fine-grained control over experiment telemetry collection, adding Experiment.telemetry.enable or Experiment.telemetry.disable to a driver script also configures experiment-level telemetry. ~ when you say fine-grained control: what do you mean? In what cases would I want to set telemetry in the driver script instead of setting via env var?

Reply:
It's possible that a user doesn't want telemetry enabled for every single experiment, so we're giving them the option of picking and choosing with more granularity.

Reply:
I see!

Reply:
what about: For workflows involving multiple experiments, SmartSim provides the attributes Experiment.telemetry.enable and Experiment.telemetry.disable to manage the enabling or disabling of telemetry on a per-experiment basis.

Reply:
I like it! Adding!